kuksa_databroker/databroker/tests/features/read_write_values.feature

```diff
@@ -1,46 +1,75 @@
 Feature: Reading and writing values of a VSS Data Entry
 
+  Rule: Access with right permissions succeeds and fails with wrong/no permissions
+
+    Background:
+      Given a running Databroker server with authorization true with the following Data Entries registered
+        | path                                    | data type | change type | type      |
+        | Vehicle.Speed                           | float     | Static      | Attribute |
+
+    Scenario: Write the current value of an unset Data Entry without authenticating fails
+      When a client sets the current value of Vehicle.Width of type float to 13.4 with token ""
+      Then the current value for Vehicle.Width can not be accessed because we are unauthorized
+
+    Scenario: Read the current value of an unset Data Entry without authenticating fails
+      When a client gets the current value of Vehicle.Width with token ""
+      Then the current value for Vehicle.Width can not be accessed because we are unauthorized
+
+    Scenario: Write the current value of a Data Entry without right permissions fails
+    """
+      The token comes from kuksa.val/jwt/read-vehicle-speed.token. Expire date is Thursday, 1 January 2026
+    """
+      When a client sets the current value of Vehicle.Speed of type float to 13.4 with token eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJsb2NhbCBkZXYiLCJpc3MiOiJjcmVhdGVUb2tlbi5weSIsImF1ZCI6WyJrdWtzYS52YWwiXSwiaWF0IjoxNTE2MjM5MDIyLCJleHAiOjE3NjcyMjU1OTksInNjb3BlIjoicmVhZDpWZWhpY2xlLlNwZWVkIn0.QyaKO-vjjzeimAAyUMjlM_jAvhlmYVguzXp76jAnW9UUiWowrXeYTa_ERVzZqxz21txq1QQmee8WtCO978w95irSRugTmlydWNjS6xPGAOCan6TEXBryrmR5FSmm6fPNrgLPhFEfcqFIQm07B286JTU98s-s2yeb6bJRpm7gOzhH5klCLxBPFYNncwdqqaT6aQD31xOcq4evooPOoV5KZFKI9WKkzOclDvto6TCLYIgnxu9Oey_IqyRAkDHybeFB7LR-1XexweRjWKleXGijpNweu6ATsbPakeVIfJ6k3IN-oKNvP6nawtBg7GeSEFYNksUUCVrIg8b0_tZi3c3E114MvdiDe7iWfRd5SDjO1f8nKiqDYy9P-hwejHntsaXLZbWSs_GNbWmi6J6pwgdM4XI3x4vx-0Otsb4zZ0tOmXjIL_tRsKA5dIlSBYEpIZq6jSXgQLN2_Ame0i--gGt4jTg1sYJHqE56BKc74HqkcvrQAZrZcoeqiKkwKcy6ofRpIQ57ghooZLkJ8BLWEV_zJgSffHBX140EEnMg1z8GAhrsbGvJ29TmXGmYyLrAhyTj_L6aNCZ1XEkbK0Yy5pco9sFGRm9nKTeFcNICogPuzbHg4xsGX_SZgUmla8Acw21AAXwuFY60_aFDUlCKZh93Z2SAruz1gfNIL43I0L8-wfw
```

Comment:
instead of specifying the (signed?) token in the feature, I would suggest to specify the authorities that the token should include and then create a corresponding token in the binding. That way we do not need to update all of the features if we decide to change the token format and/or the key used to sign the tokens.

Reply:
Seems like a reasonable adoption. Will check how this would look like.